### ISSUE TITLE
Gantry 5.5.15 is breaking the Menu Manager in Default or Maximum Error level (Joomla)

### ISSUE THREAD

**Issue by uGE70** [gantry/gantry5#3073]
Platform (MAMP Pro 6.6.2, Mac OS 12.5.1):
Joomla 4.1.5 or greater, Gantry 5.5.15-dev, Free and Premium Gantry templates, PHP 8.1.1 or greater, no cache, no SEO plugin.

**Steps to reproduce**
Using Joomla 4.1.5, set Error Reporting to default/maximum in global configuration and PHP to 8.1.1 or greater
Go to Gantry backend or any frontend page using Gantry template styles and click on the Menu tab. 

**Actual result**
I get the following error message without the actual menu being displayed:
Invalid menu structure
Menu on Frontend is working.

**How to temporarily fix the issue**
Set error level to Simple or None.

**Misc**
Tested on 4 different websites using different Gantry templates: Hydrogen, Studius, Fluent and Callisto. I guess some error notices are polluting the data streams, so I remark that a deprecated notice for the Pimple library is displayed everywhere when error level is set to maximum.
I didn't have time to test on a "fresh install", I will edit this message in a few minutes, after I'll make one.

<img width="1420" alt="Capture d’écran 2022-08-24 à 22 48 53" src="https://user-images.githubusercontent.com/3645531/186659303-25a783c7-22c4-43bc-9c70-245dd5a24c6d.png">
<img width="1406" alt="Capture d’écran 2022-08-23 à 10 14 39" src="https://user-images.githubusercontent.com/3645531/186659353-0cc119ac-e526-457b-a01c-0d79181852dc.png">

**Comment by kosh2323** [comment]
It looks like the updates haven't changed anything, PHP 8.1.9 Joomla 4.2.2 On the local OpenServer.

![13](https://user-images.githubusercontent.com/82117170/188630462-95e2cd43-f54b-4aba-a365-b1eb771b934f.jpg)

**Comment by mahagr** [comment]
Yeah, the issue is PHP 5 support. Many libraries, including Pimple, now require PHP 7.2.5 or later. I'll check if there's something I can do quickly to fix this.

In the meantime, just disable error reporting for deprecated errors and you should be fine.

**Comment by mahagr** [comment]
Can you guys try the lastest CI build?

**Comment by uGE70** [comment]
Thanks for your new build. The Pimple librairies errors have disappeared. I still get the Invalid menu structure error when PHP error reporting is set to E_ALL or Joomla error reporting is set to Maximum (I can live with that). On wordpress, the issue seems to be fixed.

**Comment by kosh2323** [comment]
> Can you guys try the lastest CI build?

When installing joomla-pkg_gantry5_develop, no errors and warnings are displayed, but when installing the joomla-tpl_g5_helium_develop template, the same warnings and errors are displayed.

![18](https://user-images.githubusercontent.com/82117170/189182322-8912a370-fd5b-4801-9f2c-5288419dc169.jpg)

![19](https://user-images.githubusercontent.com/82117170/189182343-26d298d5-6450-4bf9-89b5-105c2682b6bc.jpg)
